war/src/main/less/abstracts/theme.less

```diff
@@ -1,4 +1,4 @@
-:root {
+:root, .app-theme-picker__picker[data-theme=none] {
```

Comment:
Probably should have a comment indicating why this exists? This looks like one of those random undocumented things that'll be gone the next time someone restructures something.